Makefile

```diff
@@ -0,0 +1,21 @@
+# Help
+.SILENT:
+.PHONY: help
+
+help: ## Display this help
+	@grep -E '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
+
+# Docker
+docker-up: ## start docker elasticsearch server
+	docker-compose up -d elasticsearch
+docker-down: ## start docker elasticsearch server
+	docker-compose down
+
+# Testing
+test: test-unit test-behavior ## launch full test suite
+test-unit: ## launch unit tests
+	./vendor/bin/atoum
+test-behavior: docker-up ## launch behavior tests
+	sleep 10
```

Comment:
I think this sleep is for waiting ES instance start.

Maybe we can find an another way to check if the ES instance has started.

Reply:
You're right. This is even more relevant since I ran a test where ES server took more than 10 seconds to boot (and crashed the test) while developing the fix.